test/visual/index.html

```diff
@@ -5,15 +5,29 @@
   <title></title>
   <script src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
   <link href="../../bower_components/polymer/polymer.html" rel="import">
-  <link href="../../vaadin-element.html" rel="import">
+  <link href="../../vaadin-button.html" rel="import">
 </head>
 
 <body>
 
-  <span id="visual-tests">
-    <button>
-      Go Vaadin!
-    </button>
-  </span>
+  <form style="display: inline-block">
```

Comment:
`display: block` will take all screen width, we don't need that white space to test.